api/db/changelog/db.changelog-66-compliance-training-expiration.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="gjuggler" id="db.changelog-66-compliance-training-expiration">
+        <addColumn tableName="user">
+            <column name="compliance_training_expiration_time" type="datetime"/>
```

Comment:
Question: Will we be removing the existing training_expiration column?

Reply:
Good point – I added in code to remove the existing column.